src/candela/util/test/util.js

```diff
@@ -0,0 +1,9 @@
+import test from 'tape-catch';
+import { inferAll } from '..';
+
+test('inferAll()', t => {
+  t.deepEqual(inferAll([{'a.b': 1}]), {'a.b': 'integer'}, 'should work on fields with nested dots');
+  t.deepEqual(inferAll([{'a': {'b': 1}}]), {'a': 'string'}, 'should only accept top-level fields');
+
```

Comment:
Could we maybe add some more tests to show that this behaves the way the old function worked that it's replacing?